lusidtools/lpt/dfq.py

```diff
@@ -193,6 +194,9 @@ def load_frame(path):
                     elif df[col].dtype == int:
                         v = int(v)
                         dflt = 0
+                    elif df[col].dtype == np.int64:
+                        v = int(v)
+                        dflt = 0
```

Comment:
This is a sensible inclusion, but out of interest, what was it that prompted you to add this?

Reply:
This was causing two tests to fail locally - unclear if the test was being run in the pipeline